lib/workers/repository/update/pr/changelog/api.ts

```diff
@@ -1,9 +1,11 @@
+import { BitbucketChangeLogSource } from './bitbucket/source';
```

Comment:
we should make those lazy instantiations in a follow up PR

Reply:
FYI @setchy